src/main/scala/ai/privado/exporter/DataflowExporter.scala

```diff
@@ -65,7 +68,8 @@ class DataflowExporter(dataflowsMap: Map[String, Path], taggerCache: TaggerCache
             dataflowBySourceEntrySet._2,
             sinkSubCategory,
             ruleCache,
-            appCache
+            appCache,
+            dataflowsMap ++ extraFlowMap
```

Comment:
what is extraFlowMap?

Reply:
That map contains flows other than the flows generated by the dataflow engine.